bambi/interpret/logs.py

```diff
@@ -0,0 +1,32 @@
+import logging
+
+from bambi.interpret import logger
+
+
+def log_interpret_defaults(func):
+    interpret_logger = logger.get_logger()
+    interpret_logger.setLevel(logging.INFO)
```

Comment:
I also think you need this

https://github.com/bambinos/bambi/blob/d001e069620a494bde730d69b47fb4c5ad3ba3e6/bambi/__init__.py#L32-L36

I don't remember the details of when it was added. But I think it makes sure the messages are sent once to the place where you want to send them.